messaging/src/main/java/org/axonframework/commandhandling/SimpleCommandHandlingComponent.java

```diff
@@ -82,39 +81,33 @@ public SimpleCommandHandlingComponent subscribe(@Nonnull QualifiedName name,
 
     @Override
     public SimpleCommandHandlingComponent subscribe(@Nonnull CommandHandlingComponent commandHandlingComponent) {
-        requireNonNull(commandHandlingComponent, "The command handling component may not be null");
-        subComponents.add(commandHandlingComponent);
+        subComponents.add(requireNonNull(commandHandlingComponent, "The command handling component may not be null"));
```

Comment:
We have different preferences for this.
I prefer splitting the `requireNonNull` from the actual argument, as the amount of parentheses can become cumbersome to read. 
I know we *can* do it in one go, but I think it's more readable if we split this.